react/yarn.lock

```diff
@@ -15735,7 +15582,7 @@ remark-parse@8.0.3:
     parse-entities "^2.0.0"
     repeat-string "^1.5.4"
     state-toggle "^1.0.0"
-    trim "0.0.1"
+    trim "0.0.3"
```

Comment:
@vahidbazzaz isn't the first fixed version 3.0.1, not 0.0.3?

Reply:
@aerinsol  It appears from the [dependent bot message](https://github.com/boxwise/boxtribute/security/dependabot/36) that the first fixed version of `trim` is `0.0.3`. I think you are referring to `trim-newline`, which I corrected by upgrading dependent packages so that they already included the patch and as you mentioned, the fixed version of `trim-newline` is `3.0.1`.